Makefile

```diff
@@ -4,7 +4,7 @@ all:
 	@awk -F: '/^[[:alpha:]].*:/ && !/^all:/ {print $$1}' Makefile
 
 lint:
-	SKIP=black-format pre-commit run --all-files --hook-stage manual
+	SKIP=pre-commit run --all-files --hook-stage manual
```

Comment:
This command originally ran `pre-commit` with the environment variable named `SKIP` placed in its environment with the value `black-format`. The change here runs a command `run` with the environment variable `SKIP` placed in its environment with the value `pre-commit`.

```text
(.venv) ek@Glub:~/repos-wsl/GitPython (ruff/black $=)$ make lint
SKIP=pre-commit run --all-files --hook-stage manual
/bin/sh: 1: run: not found
make: *** [Makefile:7: lint] Error 127
```

That is not intended, since there is no `run` command, and the goal of the change here is to run `pre-commit`, just without referring to `black`-related hooks that are being removed. We can just do:

```suggestion
	pre-commit run --all-files --hook-stage manual
```

I think it would alternatively be fine to remove the `lint` target entirely from the `Makefile`. As noted in #1868, I likely should not have added it, I don't use it and I suspect nobody else does either, and the situation I added it to ease is basically not a problem anymore with `ruff` replacing all of `flake8`, `flake8`'s various plugins, and `black`. However, the minimal correctness fix in the context of this PR is as suggested above, and the target can always be removed later.

Reply:
ok, dropped lint and updated readme accordingly